Gemfile

```diff
@@ -93,3 +95,8 @@ gem 'groupdate'
 
 # Pagination
 gem 'kaminari', '~> 1.2'
+
+# Encrypt attrs
+gem 'attr_encrypted'
+
+gem 'data_migrate', '~> 9.2.0'
```

Comment:
this is pinned to `9.2.0` because of an issue with >= 9.3.0
See https://github.com/ilyakatz/data-migrate/issues/302